dyn_em/module_advect_em.F

```diff
@@ -10751,31 +10770,32 @@ PROGRAM feeder
                                     its, ite, jts, jte, kts, kte )
          ELSE IF (config_flags%scalar_adv_opt .EQ. 3 ) THEN
             CALL advect_scalar_weno ( field(ims,kms,jms,im), &
-                                   field_old(ims,kms,jms,im), &
-                                   tendency(ims,kms,jms), &
-                             ru, rv, rom, c1, c2,           &
-                             mut, time_step, config_flags,  &
-                             msfux, msfuy, msfvx, msfvy,    &
-                             msftx, msfty,                  &
-                             fzm, fzp,                      &
-                             rdx, rdy, rdzw,                &
-                             ids, ide, jds, jde, kds, kde,  &
-                             ims, ime, jms, jme, kms, kme,  &
-                             its, ite, jts, jte, kts, kte  )
+                                      field_old(ims,kms,jms,im), &
+                                      tendency(ims,kms,jms), &
+                                      ru, rv, rom,                   &
```

Comment:
Lou,
Missing c1 and c2

Reply:
Fixed.  Our network to the building is down tonight.  If the 27 April test case ran with this code - then aside from the issues you find we are good.  I have already checked the MPI 1 vs node test and it passed.